src/Window.cpp

```diff
@@ -1279,9 +1287,41 @@ int Window::mazeHeight() {
     return m_maze->getHeight();
 }
 
-bool Window::wallFront() {
+bool Window::wallFront(bool useDistance) {
```

Comment:
My guess is that we don't actually need to make any changes to this function, assuming that we move forward as far as possible as described in the comment below.